include/swift/AST/DiagnosticsSema.def

```diff
@@ -836,6 +836,11 @@ ERROR(argument_out_of_order_unnamed_unnamed,none,
       "unnamed argument #%0 must precede unnamed argument #%1",
       (unsigned, unsigned))
 
+ERROR(member_shadows_global_function,none,
+      "use of %1 refers to %0 %1 rather than %2 in %3 %4",
```

Comment:
I think this should use the full name for the member, so "use of %0 refers to %1 %2 rather than %3 in %4 %5".